.github/workflows/update-gardenctl.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
 
-filteredTag=${1##*/}
-tag=${filteredTag:-v0.17.0}
-sha=${2:-8d33c751e8d32fe7fff15306c7de59cf15c45fb04e2f9abf988d3edd3f305cc4}
+tag=${1:-v0.17.0}
```

Comment:
why are you defaulting to 0.17.0?

Reply:
I have no preference on this ;) i set default 0.17.0 there because in your code the value is default to 0.17.0 so i kept it...shall i remove the default value?

Reply:
I think it makes sense to default to a more stable release if you want to keep the defaults. Otherwise, we can just error if the value is not provided.